packages/npm-packages/ruby-wasm-wasi/test-e2e/integrations/js-require-remote.spec.ts

```diff
@@ -0,0 +1,84 @@
+import fs from "fs";
+import path from "path";
+import { test, expect } from "@playwright/test";
+import { setupDebugLog, setupProxy, resolveBinding } from "../support";
+
+if (!process.env.RUBY_NPM_PACKAGE_ROOT) {
+  test.skip("skip", () => {});
+} else {
+  test.beforeEach(async ({ context }) => {
+    setupDebugLog(context);
+    setupProxy(context, (route, relativePath, mockedPath) => {
```

Comment:
Sorry for the ambiguous function name but could you keep `setupProxy` only for mocking CDN stuff and just add new Playright routing?
I think this kind of customizing injection leads to a bloat of function responsibilities.

Reply:
I stopped extending `setupProxy`.
Playwright's routing was sufficient for this test cases.
I was wondering if there were duplicate conditions that could not be handled. Thanks for the advice.

I have left the modification that made mockedPath a local variable. Please let me know if I should delete it. I will fix it.

Reply:
Sounds reasonable to me, thanks!